src/patterns/molecules/error/error.stories.js

```diff
@@ -0,0 +1,86 @@
+import error from './error.twig';
+import '../heading/heading.scss';
+import './error.scss';
+import image from './images/5xx.svg';
+
+export default {
+  title: 'Molecules|Error',
+};
```

Comment:
To disable the `centered` add-on just for the stories in this file, modify the default export like this:
```
export default {
  title: 'Molecules|Error',
  parameters: {
    centered: {
      disable: true,
    },
  },
};
```